ScoutSuite/providers/gcp/services/cloudsql.py

```diff
@@ -82,3 +76,9 @@ def _get_instance_backups(self, instance, params):
         except Exception as e:
             printError('Failed to parse backups for SQL instance %s: %s' % (instance['name'], e))
             return None
+
+    def _is_log_enabled(self, instance) :
+        return instance['settings']['backupConfiguration'].get('binaryLogEnabled')
+
+    def _is_ssl_required(self, instance):
+        return instance['settings']['ipConfiguration'].get('requireSsl')
```

Comment:
If `instance['settings']['ipConfiguration']` doesn't contain `'requireSsl'`, then `None` will be returned instead of `False`.

```suggestion
        return instance['settings']['ipConfiguration'].get('requireSsl', False)
```

Reply:
See the [doc on `dictionary.get()`](https://www.tutorialspoint.com/python/dictionary_get.htm) for more info.